packages/gui/src/constants/WalletConnectCommands.tsx

```diff
@@ -1001,6 +1001,20 @@ const walletConnectCommands: WalletConnectCommand[] = [
       },
     ],
   },
+  {
+    command: 'getPublicKey',
+    label: <Trans>Get public key</Trans>,
+    description: <Trans>Requests a master public key from your wallet</Trans>,
+    service: ServiceName.DAEMON,
+    bypassConfirm: true,
```

Comment:
This needs to be removed. The documented feature request states that this command should always prompt the user.